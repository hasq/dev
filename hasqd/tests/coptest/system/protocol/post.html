<html>
<form method=POST action="http://127.0.0.1:13131">
<input type='text' name='var1' value='file /'/>
<input type='text' name='var2' value='bye'/>
<textarea name='area'>Some text &
new line
</textarea>
<input type='submit' value='send'/>
</form>
</html>
