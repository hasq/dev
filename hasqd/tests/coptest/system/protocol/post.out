POST / HTTP/1.1
Host: 127.0.0.1:13131
User-Agent: Mozilla/5.0 (Windows NT 6.1; WOW64; rv:25.0) Gecko/20100101 Firefox/25.0
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
Accept-Language: en-gb,en;q=0.5
Accept-Encoding: gzip, deflate
Connection: keep-alive
Content-Type: application/x-www-form-urlencoded
Content-Length: 58

var1=hello&var2=bye&area=Some+text+%26%0D%0Anew+line%0D%0A